run_FastQC_and_MultiQC.sh

```diff
@@ -0,0 +1,89 @@
+#!/bin/bash -l
+# Author: Matilda Åslin (matilda.aslin@medsci.uu.se) & Monika Brandt (monika.brandt@medsci.uu.se)
+
+if [ "$1" == "-h" ] || [ "$1" == "--help" ]
```

Comment:
since you've written this very nice usage instruction, why don't you wrap its display in a function and call that function if `-h` is passed or if you miss the required parameters? just a suggestion though, it's fine as it is as well